iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/service/CorbaNamingService.java

```diff
@@ -102,6 +109,16 @@ public void start(StartContext context) throws StartException {
                         namingPOA.servant_to_reference(ns));
             }
 
+            //print nameservice IOR to specified filename so that they can be used by external applications
+            final String iorFilepath = properties.getProperty(Constants.NAMING_IOR_FILEPATH);
+            if (!(iorFilepath == null || iorFilepath.trim().length() == 0)) {
+                final SecurityManager sm = System.getSecurityManager();
+                if (sm != null) {
+                    sm.checkPermission(FILE_PERMISSION);
```

Comment:
What is the purpose of this check, why not just attempt the operation? It looks like with this check you need to grant permission for <<ALL FILES>> , instead of just permission to the specified directory.

Reply:
The check was unnecessary - I have removed it.